utils/firestore.js

```diff
@@ -447,12 +447,11 @@ const removeParticipantsDataDestruction = async () => {
             .where(dataHasBeenDestroyed, "!=", fieldMapping.yes)
             .get();
 
-        const batch = db.batch();
-
         // Check each participant if they are already registered or more than 60 days from the date of their request
         // then the system will delete their data except the stub records and update the dataHasBeenDestroyed flag to yes.
         for (const doc of currSnapshot.docs) {
             const participant = doc.data();
+            const participantId = doc.id;
```

Comment:
NIT - the document ID doesn't represent the ID of the participant

Reply:
`docId` is a better name.
This line is not needed, since `doc.id` can be directly used and is clearer.